orders/tests.py

```diff
@@ -0,0 +1,62 @@
+import datetime
+from django.utils import timezone
+from dogowner.models import DogOwner
+from daycare.models import DayCare
+from .models import Order
+from .models import StatusOptions
+import pytest
+
+NUMBER_OF_DAYS = 3
+STARTING_DATE = timezone.now()
+FINISHING_DATE = timezone.now() + datetime.timedelta(days=NUMBER_OF_DAYS)
+PRICE_PER_DAY = 100
+STATUS = StatusOptions.Pending
+
+
+@pytest.fixture
+def create_order():
+    return Order.create(dog_owner_id=DogOwner.objects.get(id=1),
+                        daycare_id=DayCare.objects.get(id=1),
+                        starting_date=STARTING_DATE,
+                        finishing_date=FINISHING_DATE,
+                        price_per_day=PRICE_PER_DAY,
+                        )
+
+
+@pytest.mark.django_db
+class TestOrderModel:
+    def test_create_order(self):
+        new_order = Order.create(dog_owner_id=DogOwner.objects.get(id=1),
+                                 daycare_id=DayCare.objects.get(id=1),
+                                 starting_date=STARTING_DATE,
+                                 finishing_date=FINISHING_DATE,
+                                 price_per_day=PRICE_PER_DAY)
+        assert new_order in Order.objects.all()
+
+    def test_delete_order(self, create_order):
+        assert create_order in Order.objects.all()
+        create_order.delete()
+        assert create_order not in Order.objects.all()
+
+    def test_approve_order(self, create_order):
+        create_order.approve_order()
+
+        assert create_order.status == StatusOptions.Approved
+        assert create_order.approval_date is not None
+
+    def test_cancel_order(self, create_order):
+        create_order.cancel_order()
+
+        assert create_order.status == StatusOptions.Canceled
+        assert create_order.cancellation_date is not None
+
+    def test_total_price_of_order(self, create_order):
```

Comment:
a better implementation would include a parametrization with multiple combinations of PRICE_PER_DAY and NUMBER_OF_DAYS, checking we get what we expected even though we tried to fail the function

Reply:
> a better implementation would include a parametrization with multiple combinations of PRICE_PER_DAY and NUMBER_OF_DAYS, checking we get what we expected even though we tried to fail the function

Could you give an example for better implementation of the test? 
I don't think I understand what you mean by "multiple combinations of PRICE_PER_DAY and NUMBER_OF_DAYS", if it mean to check the same order with right and wrong parameters, or checking several orders.